apps/web/src/components/StrikeNotice/StrikeNotice.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { Alert } from "@gc-digital-talent/ui";
+import { useFeatureFlags } from "@gc-digital-talent/env";
+
+const StrikeNotice = () => {
+  const intl = useIntl();
+  const { psacStrike } = useFeatureFlags();
+
+  if (!psacStrike) return null;
+
+  return (
+    <Alert.Root type="warning">
+      <Alert.Title>
+        {intl.formatMessage({
+          defaultMessage:
+            "The Government of Canada and the Public Service Alliance of Canada (PSAC) are currently negotiating collective agreements.",
+          id: "oF8h5+",
+          description: "Heading for the PSAC strike notice.",
+        })}
+      </Alert.Title>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "In the event of a labour disruption, referral services from this platform will be affected. Please note that your request will be responded when services resume. Thank you for your patience.",
```

Comment:
Accurate to the issue but it should be "... your request will be responded _to_ when..."

Reply:
Fixed in [update english notice grammar](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6316/commits/fcc80af88208430274c16aee0cc7927542ccc440)